CMakeLists.txt

```diff
@@ -136,9 +136,20 @@ message("Webmin: ${SSG_PRODUCT_WEBMIN}")
 
 message(" ")
 
+include(SSGCommon)
+
+# Targets 'validate' and 'zipfile' need to be added before any product because
+# they will receive dependencies from products added
+
 add_custom_target(validate)
```

Comment:
Shouldn't SSGCommon be included after you add the custom target "validate"? As per the comment?